fastlane_bot/modes/base_pairwise.py

```diff
@@ -49,10 +49,10 @@ def get_combos(
 
         """
         all_tokens = CCm.tokens()
-        flashloan_tokens_intersect = all_tokens.intersection(set(flashloan_tokens))
+        # flashloan_tokens_intersect = all_tokens.intersection(set(flashloan_tokens))
```

Comment:
this is probably wrong to remove as we want to avoid using a flashloan token we don't really know about

Reply:
Yes agreed will revert this change.
I suspect that the confusion came from the fact that if flashloan tokens are not checksummed they get dropped at this stage, will open a separate issue for that